commands/copy.js

```diff
@@ -61,6 +61,14 @@ Data from ${cli.color.yellow(source.name)} will then be transferred to ${cli.col
       host: host(attachment.addon)
     })
   }))
+
+  if (source.attachment) {
+    let credentials = yield heroku.get(`/postgres/v0/databases/${source.attachment.addon.name}/credentials`,
```

Comment:
let's only do this when it's shogun resource (aka this won't work for yobuko)

Reply:
This should work for Yobuko as well--we shipped a similar degenerate credentials API there. But if we know the plan up-front, maybe we should avoid this check, since hobby databases will never have additional credentials? I'm +0 on that.

Reply:
GUESS WHO MADE IT https://github.com/heroku/yobuko/commits/78c42881132e9c9294cc0cb2b20b036463095f7b/web/endpoints/client/postgres.rb 🤦‍♂️  I should have asked Keiko 2017.

Well in that case, yeah I'm +0 as well, it's okay to ship as is.